### PR TITLE
Features

### PR DESCRIPTION
**Pull request by vanillainanutshell**
# What was done in this PR

- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,41 +1,12 @@
...
-window.onload = () => {
-  generateTask();
-};
\ No newline at end of file
+input();
+storage();
+valueCheck();
+contentChange();
+deleteBin();
+// taskDelete();
```
Comment (mikethreels) [inline src/index.js]: - [ ] Please Do not commit old pieces of code as inline comments. They will make your project look unprofessional.

File: src/scripts/delete.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// export const taskDelete = () => {
+//     const clearButton = document.getElementById('clear');
+//     clearButton.addEventListener('click', () => {
+//         const result = task.filter((x) => x.complete === false);
+//         task.splice(0);
+//         task.push(...result);
+//         localStorage.setItem('task', JSON.stringify(task));
+//         storageContainer();
+//     });
+// };
```
Comment (mikethreels) [inline src/scripts/delete.js]: - [ ] Please Do not commit old pieces of code as inline comments. They will make your project look unprofessional.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Matt, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] please make sure you have professional readme file as requested by Microverse, have a look at [the template](https://github.com/microverseinc/readme-template) if you need an example.

## Optional suggestions
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/scripts/delete.js (hunk 2 above)
Comment (vanillainanutshell) [reply src/scripts/delete.js]: they are not old pieces of code, they are actually parts of the next requirements that I've commented

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @vanillainanutshell ,  

Your project is complete!

### OPTIONAL

- [ ] **I have noticed that your readme is empty, I strongly suggest you to update it, it can be a strong reason to invalidate the pull request.** 
- [ ] I have noticed a bug in your code, after adding items users cannot immediately delete or update them, these actions can be done only after refreshing the page. I kindly suggest to fix it 😉 

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**